### PR TITLE
Add new configs `dbms.security.http_static_content_security_policy_header`, `server.http.transaction_idle_timeout`

### PR DESCRIPTION
**Pull request by NataliaIvakina**
…ader`, `server.http.transaction_idle_timeout`

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1592.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1592.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -3901,6 +3900,22 @@ m|
...
+[[config_dbms.security.http_static_content_security_policy_header]]
+=== `dbms.security.http_static_content_security_policy_header`
+
+.dbms.security.http_static_content_security_policy_header
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Defines the Content-Security-Policy header to return to content returned on static endpoints.
+|Valid values
+a|A string
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:3913]:
```suggestion
a|A string.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1592.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1592.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 2]
```diff
@@ -3901,6 +3900,22 @@ m|
...
+
+.dbms.security.http_static_content_security_policy_header
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Defines the Content-Security-Policy header to return to content returned on static endpoints.
+|Valid values
+a|A string.
+|Default value
+m|+++default-src 'self'; script-src 'self' cdn.segment.com canny.io; img-src 'self'; style-src 'self' fonts.googleapis.com 'unsafe-inline'; font-src 'self' fonts.gstatic.com; base-uri 'none'; object-src 'none'; frame-ancestors 'none'; connect-src 'self' api.canny.io api.segment.io wss+++
```
Comment (gjmwoods) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:3915]: we just [merged an update](https://github.com/neo-technology/neo4j/pull/25301/files) for the default value:
```suggestion
m|+++default-src 'self'; script-src 'self' cdn.segment.com canny.io; img-src 'self' data:; style-src 'self' fonts.googleapis.com 'unsafe-inline'; font-src 'self' fonts.gstatic.com; base-uri 'none'; object-src 'none'; frame-ancestors 'none'; connect-src 'self' api.canny.io api.segment.io ws: wss: http: https:+++
```

**Review by gjmwoods — APPROVED** [review]
thank you! 🙇

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1592.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1592.surge.sh/operations-manual/5/configuration/configuration-settings)

**Pull request merged**